### ISSUE TITLE
Unable to compile under Linux

### ISSUE THREAD

**Issue by TCH68k** [OtterBrowser/otter-browser#1752]
I get this error message:
```
/opt/otter/src/modules/windows/cache/CacheContentsWidget.cpp: In member function ‘void Otter::CacheContentsWidget::updateActions()’:
/opt/otter/src/modules/windows/cache/CacheContentsWidget.cpp:443:23: error: ‘class Otter::Ui::CacheContentsWidget’ has no member named ‘formWidget’
  443 |  const int size(m_ui->formWidget->contentsRect().height() - 10);
      |                       ^~~~~~~~~~
make[2]: *** [CMakeFiles/otter-browser.dir/build.make:2817: CMakeFiles/otter-browser.dir/src/modules/windows/cache/CacheContentsWidget.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:96: CMakeFiles/otter-browser.dir/all] Error 2
make: *** [Makefile:171: all] Error 2
```
I built Otter three days ago without errors.

OS is Debian 11, AMD64. Compiler is GCC 10. Qt is 5.15.2.

**Comment by Emdek** [comment]
@TCH68k, fixed in 75098b8.

**Comment by TCH68k** [comment]
Confirmed, it works. Thank you.
